crates/matrix-sdk-ui/src/timeline/pagination.rs

```diff
@@ -103,7 +103,7 @@ impl super::Timeline {
     /// call to [`Self::paginate_backwards()`].
     pub async fn live_back_pagination_status(
         &self,
-    ) -> Option<(RoomPaginationStatus, impl Stream<Item = RoomPaginationStatus>)> {
+    ) -> Option<(RoomPaginationStatus, impl Stream<Item = RoomPaginationStatus> + use<>)> {
```

Comment:
Here it's fine because there is no named type we can use: `impl Stream` is mandatory.